.gitignore

```diff
@@ -44,7 +44,7 @@ scripts/prepare/results/
 packages/java/hilla/*.json
 packages/java/hilla-react/*.json
 packages/java/hilla/src/main/java/com/vaadin/hilla/theme/
-packages/java/hilla-react/src/main/java/com/vaadin/hilla/theme/
+packages/java/hilla/src/main/resources/
```

Comment:
This one looks a bit unexpected. Which files are supposed to be ignored there?

Reply:
There simply aren't any files yet other than generated ones.